### DISCUSSION TITLE
Can EntraCP be restricted to only return Groups ?

### DISCUSSION THREAD

**Discussion by NaveenKanukuntla** [Yvand/EntraCP#215] [General]
Here is my setup

1. EntraID as the directory lookup.
2. Utilizing EntraCP to fill claims for user on SharePoint 2016.
3. I already use Windows /Claims NTLM for user lookup from active directory.
4. Most of my security groups have been created in Azure and not in On-Prem AD.

Due to this, I would like to use EntraCP to lookup Security Groups in EntraID

Is it possible to limit my people picker results returned from EntraCP to only resolve/return role claims ?

**Comment by Yvand** [comment]
@NaveenKanukuntla in short, no, this is not possible.
The reason is that it is all or nothing: If you use EntraCP, it needs to be associated with the trust and will handle all the users and groups for that trust

**Reply by NaveenKanukuntla** [reply]
Thank you for your reply @Yvand
